cdap-common/src/test/java/io/cdap/cdap/common/io/CachingLocationTest.java

```diff
@@ -195,8 +195,8 @@ public void testNoDuplicateCacheEntries() throws IOException, InterruptedExcepti
     Assert.assertEquals(1, cachingPathProvider.getCacheEntries().size());
 
     // Modify the file, last modified should change, adding a new entry in the cache map.
-    // Sleep for a while to ensure at least 1 millis has passed.
-    Thread.sleep(1);
+    // Sleep for a while to ensure at least 1 second has passed.
```

Comment:
Can you mention that 1 second is the minimum resolution of the last modified time in MacOS? In the future, someone may not understand why we set 1 sec.

Reply:
Done.